Analyses/src/StoppedParticlesFinder_module.cc

```diff
@@ -88,7 +88,7 @@ namespace mu2e {
     std::string stoppingMaterial_;
     std::vector<std::string> vetoedMaterials_;
 
-    unsigned offsetThreshold_; // to select particles from the current simulation stage
+    unsigned simStageThreshold_; // to select particles from the current simulation stage
```

Comment:
I am confused why this is necessary.  Wouldn't I avoid this problem simply by passing in the StepPointMC collection that corresponds to the stage I want to resample?  I could even specify process if it's necessary.

Reply:
We are looking for stopped particles, not for step points.   SimParticleCollection includes the full history.   Depending of the details of fcl config for different stages this may or may not be important.  It is necessary for CD3-like arrangements where the charged particle beam coming to TS5 includes both muons and beam electrons.   The latter may have originated from mu stops in the upstream part of the beamline; this config is a way to avoid double-counting those mu stops.